packages/sdk/src/Configuration/Policies/NoDepegOnRedeemSharesForSpecificAssets.ts

```diff
@@ -58,19 +53,7 @@ export function encodeSettings(args: Settings): Hex {
  * @returns The decoded settings.
  */
 export function decodeSettings(encoded: Hex): Settings {
-  const [assets, referenceAssets, deviationTolerancesInBps] = decodeAbiParameters(settingsEncoding, encoded);
-
-  return {
-    assetConfigs: assets.map((asset, i) => {
-      const referenceAsset = referenceAssets[i];
-      const deviationToleranceInBps = deviationTolerancesInBps[i];
-      Assertion.invariant(referenceAsset !== undefined, "Expected referenceAssets and assets to have the same length");
-      Assertion.invariant(
-        deviationToleranceInBps !== undefined,
-        "Expected deviationToleranceInBpss and assets to have the same length",
-      );
+  const [assetConfigs] = decodeAbiParameters(settingsEncoding, encoded);
 
-      return { asset, referenceAsset, deviationToleranceInBps };
-    }),
-  };
+  return { assetConfigs };
```

Comment:
I'm not sure if wrapping assets configs into an object is necessary. I mean instead of returning `{ assetConfigs }`, we could return `assetConfigs`, imho this extra layer doesn't add anything, but increases complexity.
Same for encoding

Reply:
done